proto/cosmos/consensus/v1/consensus.proto

```diff
@@ -0,0 +1,23 @@
+// Since: cosmos-sdk 0.51
+syntax = "proto3";
+package cosmos.consensus.v1;
+
+import "tendermint/types/params.proto";
+
+option go_package = "github.com/cosmos/cosmos-sdk/x/consensus/types";
+
+// ConMsgParams is the Msg/Params request type. This is a consensus message that is sent from cometbft.
+message ConMsgParams {
```

Comment:
nit: Is there a reason to abbreviate `Consensus`?

Reply:
smaller footprint since larger names impact overall size